build-tools/packages/build-tools/src/fluidBuild/fluidPackageCheck.ts

```diff
@@ -149,10 +149,30 @@ export class FluidPackageCheck {
     ) {
```

Comment:
I'm kind of wondering about the structure of this method. The `expected` parameter no longer really has any meaning for our lint scripts. I wonder if extracting our lint checks out as a separate `checkLintScripts` method might be a bit better, so we can don't have unused parameters and can document the correct expectations? What do you think?